app/assets/stylesheets/application/publishers.sass

```diff
@@ -259,7 +259,6 @@ body[data-controller="u2f_registrations"]
     .info-panel
       padding-top: 100px
     .brave-logo-large
-      background: url(asset-path("brave_icon_512x.png")) 0 0 / 128px 128px no-repeat
```

Comment:
Should we also remove `/app/assets/images/brave_icon_512x.png` from the repo?

Reply:
Done! I was being overly paranoid in leaving it.